src/style.css

```diff
@@ -1 +1,45 @@
-/* write your styles here */
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  margin: auto;
+  width: 300px;
+  height: 100px;
+}
+
+#header {
+  background-color: blue;
+}
+
+#footer {
+  background-color: black;
+}
+
+.important {
+  background-color: red;
+}
+
+.interesting {
+  background-color: green;
+}
+
+.surprising {
+  background-color: orange;
+}
+
+#header:hover,
+#footer:hover,
+.important:hover,
+.interesting:hover,
+.surprising:hover {
```

Comment:
add class block for each element, and add this styles and styles at lines 9-16 to it